src/TLuaInterpreter.cpp

```diff
@@ -6974,7 +6974,7 @@ int TLuaInterpreter::permSubstringTrigger( lua_State * L )
 
 int TLuaInterpreter::permKey( lua_State *L )
 {
-    quint8 argIndex = 0;
+    uint_fast8_t argIndex = 0;
```

Comment:
It wasn't intended as a "work_around" it was just that I anticipated the maximum number of bits and was willing to be explicit about it instead of just having the generic `int` and the Qt types are well defined (and will likely now be `typedef`ed to the  longer typename of C++1x auto-magically anyhow)...

As it happened I was going to change the `keyModifier` and `keyCode` to be `quint64` rather than `int`as I remember that the Qt Clang-analyser plugin does not like assigning the value of `lua_tointeger(...)`to it but that would be disruptive as it means changing some reference passing of arguments to other functions.

_(I have the analyser switched off currently as it is a CPU hog but it gets upset because `lua_interpreter(...)` returns a `long` rather than an `int` on some/all platforms and that is iffy on Windows IIRC...)_